src/eventhubs/azext_eventhub/_exception_handler.py

```diff
@@ -0,0 +1,16 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.util import CLIError
+
+
+def eventhubs_exception_handler(ex):
+    from azext_eventhub.eventhub.models import ErrorResponseException
+    if isinstance(ex, ErrorResponseException):
+        raise CLIError(ex.message)
+    else:
+        import sys
+        from six import reraise
+        reraise(*sys.exc_info())
```

Comment:
We expect an empty result on show commands, lot an exception to be raised.
See https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/util.py#L36-L40
Can you just use that method directly?

Reply:
added method with ErrorResponseException, that method supports clouderror exception